client/src/pages/Login.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { Button, TextField, Typography, makeStyles } from "@material-ui/core";
+import SignUpContainer from "../components";
+
+const useStyles = makeStyles({
+  input: {
+    textAlign: "center",
+    width: "60%",
+    margin: "2vh"
+  },
+  text: {
+    fontSize: "3vw",
+    fontWeight: "800"
+  },
+  button: {
+    backgroundColor: "#43DDC1"
+  },
+  link: {
+    marginTop: "2vh"
+  }
+});
+
+const Login = () => {
+  const classes = useStyles();
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  const submit = () => {
+    return; // placeholder
+  };
+
+  return (
+    <SignUpContainer>
+      <Typography className={classes.text}> Sign In </Typography>
+      <br />
```

Comment:
Let's avoid using breaklines, it's generally bad practice. Instead we use padding or margin. In Typography there is paragraph + gutterBottom